spec/models/concerns/sluggable.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+
+shared_examples_for 'sluggable' do
+  let(:factory_name) { described_class.name.parameterize.tr('-', '_').to_sym }
+  let(:attr_name) { "#{described_class.name} Second Third" }
+
+  describe 'generate_slug' do
+    before do
+      create(factory_name, name: attr_name)
```

Comment:
I'm using the `name` default attr here :( seemed overkill to find my way around the possible overwritting of the attr used for the slugs... so when the time comes we'll figure it out, or if it's a fork.. they'll find their way for sure :)